Cargo.toml

```diff
@@ -35,17 +36,20 @@ arbitrary = { version = "1.4.1", default-features = false, features = [] }
 arrayvec = { version = "0.7.6", default-features = false, features = [] }
 bindgen = { version = "0.71.1", default-features = false, features = [] }
 bolero = { version = "0.12.0", default-features = false, features = [] }
-clap = { version = "4.5.27", default-features = false, features = ["std"] }
+bytes = { version = "1.9.0", default-features = false, features = [] }
+clap = { version = "4.5.30", default-features = false, features = ["std"] }
 ctrlc = { version = "3.4.5", default-features = false, features = [] }
 doxygen-rs = { version = "0.4.0", default-features = false, features = [] }
 etherparse = { version = "0.17.0", default-features = false, features = [] }
+ipnet = { version = "2.11.0", default-features = false, features = [] }
+iptrie = { version = "0.10.2", default-features = false, features = [] }
```

Comment:
I'm fine to upgrade this one, but 0.10.2 has been very recently released. I wonder how should we be dealing in those cases when it comes to upgrades, given that our coverage isn't high at the moment?

Reply:
I don't have too much input here, but iptrie v0.9 has been released in December 2024 only (so, not long ago) and doesn't seem to have a huge amount of users, judging by the number of contributors and GitHub stars - for what it's worth. So I don't think we're taking a lot more risks here.